systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,389 @@
+                .build();
+        resourceManager.createResource(extensionContext, true, adminClientJob.defaultAdmin().build());
+        String createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(
+                createPodName,
+                "org.apache.kafka.common.errors.ThrottlingQuotaExceededException: The throttling quota has been exceeded.",
+                Duration.ofMinutes(5).toMillis()
+        );
+        JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+
+        // Teardown delete created topics
+        List<KafkaTopic> ktl = KafkaTopicUtils.getAllKafkaTopicsWithPrefix(NAMESPACE, topicNamePrefix);
+        for (KafkaTopic kt : ktl) {
+            cmdKubeClient().deleteByName(KafkaTopic.RESOURCE_SINGULAR, kt.getMetadata().getName());
+        }
+    }
+
+    @ParallelTest
+    void testThrottlingQuotasDeleteTopic(ExtensionContext extensionContext) {
+        final String kafkaUsername = kafkaUsernamePrefix + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String createAdminName = "create-admin-client" + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String deleteAdminName = "delete-admin-client" + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String topicNamePrefix = classTopicPrefix + "-delete";
+        int topicsCountOverQuota = 500;
+        setupKafkaUserInNamespace(extensionContext, kafkaUsername);
+
+        // Create many topics in multiple rounds using starting offset
+        KafkaAdminClient createAdminClientJob;
+        String createPodName;
+        int offset = 100;
+        int iterations = topicsCountOverQuota / 100;
+        for (int i = 0; i < iterations; i++) {
+            LOGGER.info("Executing {}/{} iteration.", i + 1, iterations);
+            createAdminClientJob = new KafkaAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(100)
+                .withTopicOffset(i * offset)
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.CREATE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername))
+                .build();
+            resourceManager.createResource(extensionContext, true, createAdminClientJob.defaultAdmin().build());
+            ClientUtils.waitForClientSuccess(createAdminName, NAMESPACE, 10);
+            createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+            PodUtils.waitUntilMessageIsInPodLogs(createPodName, "All topics created", Duration.ofMinutes(6).toMillis());
+            JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+        }
```

Comment:
Again, why don't you create `KafkaAdminClient` with all parameters, that are same and then just do the `toBuilder()` and replace stuff that differs? This applies to all clients in this class, not just one for loop.